kernel/data-pipeline/cdc/client/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/client/importer/DataSourceImporter.java

```diff
@@ -57,7 +60,47 @@ public DataSourceImporter(final String databaseType, final ImportDataSourceParam
     }
     
     @Override
-    public void write(final Record record) throws Exception {
+    public void write(final List<Record> records) throws Exception {
```

Comment:
Does `records` belong to the same table? If not, then need to group by table firstly